### DISCUSSION TITLE
Media queries / Responsiveness

### DISCUSSION THREAD

**Discussion by cwagner22** [vikejs/vike#1972] [Discussion]
I have a react page that is using useMediaQuery (mui) to render a different component on mobile.
Is there any recommended way to handle SSR responsiveness?

**Comment by brillout** [comment]
Feel free to elaborate if you believe your question is related to Vike. ([The general expectation on Vike's GitHub discussion is to post questions related to Vike](https://vike.dev/faq#github).)
